### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by llj110** [Kki2Eve/RISNet#4]
Is the training parameters of other COD datasets consistent with those of ACOD-12K? Can you provide the loading method and training parameters for these COD datasets?

**Comment by Kki2Eve** [comment]
Thank you for your message. We made slight adjustments to the training parameters on other COD datasets, increasing the batch size from 4 to 8. This change was made to fully utilize the GPU memory since there were no depth maps in the other datasets. Similarly, the loading method for these COD datasets was implemented by removing the depth information loading in dataloader.py. I hope this clarifies your question.

**Comment by llj110** [comment]
Thanks for your valuable reply. Here, I have an extra question: For the dataset of NC4K, at what ratio did you divide the training and testing sets?

**Comment by Kki2Eve** [comment]
The entire NC4K dataset is treated as part of the test set.

**Comment by llj110** [comment]
Do you mean to combine the training set of COD10K and CAMO to train a model, and then test it separately on the testing sets of COD10K and CAMO, as well as NC4K?

**Comment by Kki2Eve** [comment]
Yes, that's it.

**Comment by LingShaQ** [comment]
Hello author, sorry to bother you,when training ACOD-12K on SiNet, MGL, SINet-V2 and other models, do you use the Imgs and GT in the data set, but not the Depth of the ACOD-12K data set?

**Comment by LingShaQ** [comment]
Hello author, I'm sorry to bother you. When training ACOD-12K on SiNet, MGL, SINet-V2 and other models, SiNet and other models use Imgs and GT in the ACOD-12K data set, but do not use the ACOD-12K data set. depth.Is this how I understand it?

**Comment by yldance** [comment]
> 作者您好，很抱歉打扰您，在SiNet、MGL、SINet-V2等模型上训练ACOD-12K时，SiNet等模型使用了ACOD-12K数据集中的Imgs和GT，但没有使用ACOD-12K数据集的深度。是我理解的吗？

Sinet只用是rgb图像，你不更改dataloader的话是不会用到depth的
